spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.